openproblems/tasks/_batch_integration/batch_integration_graph/README.md

```diff
@@ -0,0 +1,46 @@
+<!--- TODO: add links --->
+
+# Batch integration graph
+
+This is a sub-task of the overall batch integration task. Batch (or data) integration methods integrate datasets across batches that arise from various biological and technical sources. Methods that integrate batches typically have three different types of output: a corrected feature matrix, a joint embedding across batches, and/or an integrated cell-cell similarity graph (e.g., a kNN graph). This sub-task focuses on all methods that can output integrated graphs, and includes methods that canonically output the other two data formats with subsequent postprocessing to generate a graph. Other sub-tasks for batch integration can be found for:
+
+* [embeddings](../batch_integration_embed/), and
+* [corrected features]()
+
+This sub-task was taken from a [benchmarking study of data integration methods](https://www.biorxiv.org/content/10.1101/2020.05.22.111161v2).
+
+
+## API
+
+Datasets should contain the following attributes:
+
+* `adata.obs["batch"]` with the batch covariate,
+* `adata.obs["label"]` with the cell identity label,
+* `adata.layers['counts']` with raw, integer UMI count data, and
+* `adata.obsp['uni_connectivities']` with an unintegrated connectivity matrix generated
+  by  `scanpy.pp.neighbors()`
+* `adata.X` with log-normalized data
```

Comment:
need to add `adata.obsm['X_uni']` as the PCA embedding of the unintegrated representation here.